src/main/java/com/fakng/fakngagrgtr/enumiration/FakngError.java

```diff
@@ -0,0 +1,45 @@
+package com.fakng.fakngagrgtr.enumiration;
+
+public enum FakngError {
+
+
+    VACANCY_NOT_FOUND("011", "Vacancy not found."),
```

Comment:
I'd expect those messages to be rather formats. Because these are not really descriptive. When a vacancy is not found, I wanna know which one exactly. Because if I see something like this in logs, it doesn't help me to identify where to look at

Reply:
I can add parameters that will need to be added to the exception to create it. (for example, id, title or location)

Reply:
Do it?

Reply:
@mighty-mhsl 
Can we call or write to resolve the issue of formats? I see several options:
1) Creating formatted strings for id, name, location, etc
2) Creating a standard formatted string in which you can specify any field and any value
3) Creating formatted strings according to some requirements, a group of fields, etc

What to choose from this?
Is there anything else you can offer?

Reply:
Rather each string should have placeholders for the appropriate fields

Reply:
Well